lib/index.d.ts

```diff
@@ -17,7 +17,8 @@ declare namespace VueLoader {
     cacheDirectory?: string
     cacheIdentifier?: string
     prettify?: boolean
-    exposeFilename?: boolean
+    exposeFilename?: boolean,
+    filterStyleTag?: (styleTagAttributes: Object) => boolean,
```

Comment:
I typed it as an Object even though it is a `ParsedUrlQuery` internally (users don't need to know its real interface in my opinion, but it is poorly typed with just `object`).   
I'm opened to types suggestions here!